src/templates/Front/docs.md

```diff
@@ -177,5 +177,95 @@ Von [Christof Moser](/~6556e282-4ac4-4129-8f8f-d20f28170c39 "Christof Moser")
 
 <hr /></section>
 
+<section><h6>TEASER</h6>
+
+\`\`\`
+{
+  "teaserType": "frontArticleCollection",
+  "url": "https://www.republik.ch/updates/wer-sind-sie"
+}
+\`\`\`
+
+<section><h6>ARTICLECOLLECTIONINTRO</h6>
```

Comment:
This is now more like the teaser. For the teaser module to just work this probably can't be nested in another zone. We probably have to restructure this into:

```
{
  dossierZone (section DOSSIER)
  rules: [
    dossierTeaser (section TEASER)
    articleTileRow (section TEASERGROUP)
  ]
}
```

Let's double check with @lukasbuenger